internal/rootcoord/create_collection_task.go

```diff
@@ -76,6 +75,30 @@ func (t *createCollectionTask) validate() error {
 		return fmt.Errorf("shard num (%d) exceeds system limit (%d)", shardsNum, cfgShardLimit)
 	}
 
+	db2CollIDs := t.core.meta.ListAllAvailCollections(t.ctx)
+	totalCollections := 0
+	for _, collIDs := range db2CollIDs {
+		totalCollections += len(collIDs)
+	}
+
+	maxCollectionNum := Params.QuotaConfig.MaxCollectionNum
+	if totalCollections >= maxCollectionNum {
+		log.Warn("unable to create collection because the number of collection has reached the limit", zap.Int("max_collection_num", maxCollectionNum))
```

Comment:
Maybe `info` here is better, anyway this error will be seen by the user.

Reply:
This is an unexpected log, it should not be info level.